inferentia/qa/setup_test_enviroment_and_test.sh

```diff
@@ -34,22 +34,24 @@ export TRITON_THIRD_PARTY_REPO_TAG=${DEFAULT_REPO_TAG}
 export IDENTITY_BACKEND_REPO_TAG=${DEFAULT_REPO_TAG}
 export PYTHON_BACKEND_REPO_TAG=${DEFAULT_REPO_TAG}
 export CHECKSUM_REPOAGENT_REPO_TAG=${DEFAULT_REPO_TAG}
-export TRITON_SERVER_BRANCH_NAME=${TRITON_SERVER_BRANCH_NAME:=${DEFAULT_REPO_TAG}}
+export TRITON_SERVER_REPO_TAG=${TRITON_SERVER_REPO_TAG:=${DEFAULT_REPO_TAG}}
 export TRITON_CLIENT_REPO_TAG=${TRITON_CLIENT_REPO_TAG:=${DEFAULT_REPO_TAG}}
-export TRITON_VERSION="2.17.0dev"
-export TRITON_CONTAINER_VERSION="21.12dev"
-export TRITON_UPSTREAM_CONTAINER_VERSION="21.10"
 export BASE_IMAGE=tritonserver
 export SDK_IMAGE=tritonserver_sdk
 export BUILD_IMAGE=tritonserver_build
 export QA_IMAGE=tritonserver_qa
+export TEST_JSON_REPO=/opt/tritonserver/qa/common/inferentia_perf_analyzer_input_data_json
+export TEST_REPO=/opt/tritonserver/qa/L0_inferentia_perf_analyzer
+export TEST_SCRIPT="test.sh"
+CONTAINER_NAME="qa_container"
+
 
 cd ${TRITON_PATH}
+echo $TRITON_SERVER_REPO_TAG
```

Comment:
Remove this

Reply:
I think it's helpful for the user to read the server repo tag.. I imagine it to be frequent that the user will need to input the server repo that's not "main"

Reply:
It is not useful to just echo a branch name. Echo a more readable message such as `server_repo_tag: <branch>`